examples/jsm/nodes/ShaderNode.js

```diff
@@ -186,7 +186,7 @@ export const nodeObject = ( val ) => {
 
 export const uniform = ( constNode ) => {
 
-	return nodeObject( new UniformNode( constNode.value, constNode.nodeType ) );
+	return nodeObject( new UniformNode( constNode.value?.clone?.() || constNode.value, constNode.nodeType ) );
```

Comment:
Why did you need to clone this class?

Reply:
I think it can be removed, I have added it because of cache.